implementations/rust/ockam/ockam_identity/src/identities/storage/sqlite_storage.rs

```diff
@@ -0,0 +1,185 @@
+use crate::Storage;
+use core::str;
+use ockam_core::async_trait;
+use ockam_core::compat::sync::{Arc, Mutex};
+use ockam_core::compat::vec::Vec;
+use ockam_core::errcode::{Kind, Origin};
+use ockam_core::{Error, Result};
+use ockam_node::tokio::task::{self, JoinError};
+use rusqlite::{params, Connection};
+use std::fmt;
+use std::path::Path;
+use tokio_retry::strategy::{jitter, FixedInterval};
+use tokio_retry::Retry;
+use tracing::debug;
+
+/// Storage using the Sqlite database
+#[derive(Clone)]
+pub struct SqliteStorage {
+    /// Sqlite Connection
+    pub conn: Arc<Mutex<Connection>>,
```

Comment:
Could you please make it private and create a getter that would return a clone of the `Arc`?

Reply:
sure thing